.github/workflows/build.yml

```diff
@@ -296,3 +298,57 @@ jobs:
           python-version: ${{ matrix.python }}
       - run: make init
       - run: pytest -vv tests/integration/workflows/dotnet_clipackage
+
+  rust-cargo-lambda-integration:
+    name: ${{ matrix.os }} / ${{ matrix.python }} / rust-cargo-lambda
+    if: github.repository_owner == 'aws'
+    runs-on: ${{ matrix.os }}
+    env:
+      CARGO_LAMBDA_VERSION: 0.15.0
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+        python:
+          - "3.9"
+          - "3.8"
+          - "3.7"
+        rust:
+          - stable
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      # Install and configure Rust
+      - name: Install rustup
+        run: |
+          : install rustup if needed
+          if ! command -v rustup &> /dev/null ; then
+            curl --proto '=https' --tlsv1.2 --retry 10 --retry-connrefused -fsSL "https://sh.rustup.rs" | sh -s -- --default-toolchain none -y
+            echo "${CARGO_HOME:-$HOME/.cargo}/bin" >> $GITHUB_PATH
+          fi
+        if: ${{ matrix.os }} == 'ubuntu-latest'
+      - name: rustup toolchain install ${{ matrix.rust }}
+        run: rustup toolchain install ${{ matrix.rust }} --profile minimal --no-self-update
+      - run: rustup default ${{ matrix.rust }}
+      - run: |
+          : disable incremental compilation
+          echo CARGO_INCREMENTAL=0 >> $GITHUB_ENV
+      - run: |
+          : enable colors in Cargo output
+          echo CARGO_TERM_COLOR=always >> $GITHUB_ENV
+
+      # Install and configure Cargo Lambda
+      - name: Install Cargo Lambda
+        run: pip install cargo-lambda==$CARGO_LAMBDA_VERSION
```

Comment:
Will customer require to install this if they want to use this builder?

Reply:
Do we have some guardrails in the implementation that will gracefully exit if this is not available in the customer's machine?